code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,293 @@
+			new /obj/item/clothing/head/bio_hood/virology(src)
+			new /obj/item/clothing/suit/bio_suit/virology(src)
+			new /obj/item/tank/internals/oxygen/yellow(src)
+			new /obj/item/clothing/gloves/color/latex/nitrile(src)
+		if(108)
+			new /obj/item/circuitboard/machine/protolathe(src)
+		if(109)
+			var/chosen_circuit = pick(subtypesof(/obj/item/circuitboard) - /obj/item/circuitboard)
+			new chosen_circuit(src)
+		if(110)
+			new /obj/item/clothing/head/bronze(src)
+			new /obj/item/clothing/shoes/bronze(src)
+			new /obj/item/clothing/suit/bronze(src)
+			new /obj/item/stack/tile/bronze/thirty(src)
+		if(111)
+			new /obj/item/clothing/accessory/medal/bronze_heart(src)
+			new /obj/item/clothing/accessory/medal/conduct(src)
+			new /obj/item/clothing/accessory/medal/plasma(src)
+			new /obj/item/clothing/accessory/medal/plasma/nobel_science(src)
+			new /obj/item/clothing/accessory/medal/ribbon/cargo(src)
+			new /obj/item/clothing/accessory/medal/silver/excellence(src)
+			new /obj/item/clothing/accessory/medal/silver/security(src)
+			new /obj/item/clothing/accessory/medal/silver/valor(src)
+		if(112)
+			new /obj/item/clothing/accessory/medal/gold/heroism(src)
+			new /obj/item/clothing/accessory/medal/gold/captain(src)
+			new /obj/item/reagent_containers/food/drinks/trophy/gold_cup(src)
+			new /obj/item/reagent_containers/food/snacks/grown/apple/gold(src)
+		if(113)//Lite, mid and heavy are Snowndin lootdrops. Putting a spawner into the chest result things spawn on top of it instead of inside.
+			switch(pickweight(list("heavystuff1" = 4, "heavystuff2" = 3, "heavystuff3" = 1, "heavystuff4" = 4, "heavystuff5" = 2, "heavystuff6" = 1, "heavystuff7" = 2, "heavystuff8" = 1)))
+				if("heavystuff1")
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+				if("heavystuff2")
+					new /obj/item/clothing/head/helmet/roman/legionnaire(src)
+					new /obj/item/clothing/shoes/roman(src)
+					new /obj/item/clothing/under/costume/roman(src)
+					new /obj/item/shield/riot/roman(src)
+					new /obj/item/melee/shank(src)
+				if("heavystuff3")
+					new /obj/item/twohanded/fireaxe(src)
+					new /obj/item/organ/brain/alien(src)
+				if("heavystuff4")
+					new /obj/item/modular_computer/tablet/preset/advanced(src)
+				if("heavystuff5")
+					new /obj/item/stack/spacecash/c1000(src)
+					new /obj/item/stack/spacecash/c1000(src)
+					new /obj/item/clothing/head/collectable/tophat(src)
+					new /obj/item/storage/fancy/cigarettes/cigars/cohiba(src)
+				if("heavystuff6")
+					new /obj/item/borg/upgrade/syndicate(src)
```

Comment:
Powerful antagonist items should not be in these crates considering they are easy to open and encourage meta programs